android/analytics.md

```diff
@@ -64,7 +64,7 @@ The Amplify CLI helps setup and configure Pinpoint within your application and c
 
 #### Update your IAM Policy:
 
-Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API:
+Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API. If you are using long-term AWS credentials of an `Amazon IAM` user, attach the following policies to the role of the `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`.
```

Comment:
The Amazon Pinpoint service requires permissions defined in an IAM policy to use the `submitEvents` API. If you are using long-term AWS credentials attached to an `Amazon IAM` user, attach the following policies to the role of that `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, then attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`. The role you attach the policies to depends on the scope of your application. For example, if you only want events submitted when users login, attach to the authenticated role. Similarly, if you want events submitted regardless of authentication state, attach the policy to the unauthenticated role. For more information on Cognito Identities authenticated/unauthenticated roles see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/identity-pools.html" target="_blank">here</a>.

Reply:
Changed